base/MaplGrid.F90

```diff
@@ -144,9 +135,115 @@ subroutine set(this, grid, unusable, rc)
   
     deallocate(jms, ims)
 
+    call GridCoordGet(   this%ESMFGRID, this%LATS       , &
+         Name     = "Latitude"              , &
+         Location = ESMF_STAGGERLOC_CENTER  , &
+         Units    = 99                      , & !ESMFL_UnitsRadians
```

Comment:
Could we change '99' to a predefine constant, for example ESMFL_UnitsRadians. This unfortunately needs adding 'use ESMFL_Mod' at the top. Similar change for line 147
```suggestion
         Units    = ESMFL_UnitsRadians, &
```

Reply:
Will change, commit , test, and push.